k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -660,10 +660,10 @@ function deployCortxConfigMap()
     # Generate node file with type control_node in "node-info" folder
     new_gen_file="$node_info_folder/cluster-control-node.yaml"
     cp "$cfgmap_path/templates/cluster-node-template.yaml" $new_gen_file
-    ./parse_scripts/subst.sh $new_gen_file "cortx.node.name" "cortx-control-headless-svc"
+    ./parse_scripts/subst.sh $new_gen_file "cortx.node.name" "cortx-control"
```

Comment:
I have an open question to Ujjwal about what this name is used for (if anything).  I changed the name only because it seemed weird removing the cortx-control-headless-svc Service, but to leave this node name in there.